enumerable.rb

```diff
@@ -0,0 +1,104 @@
+module Enumerable
+  def my_each
+    each do |i|
+      yield i
+    end
+  end
+
+  def my_each_with_index
+    count = 0
+    while count < length
+      yield(self[count], count)
+      count += 1
+    end
+  end
+
+  def my_select
+    filtered_array = []
+    each do |i|
+      filtered_array.push(i) if yield i
+    end
+    filtered_array
+  end
+
+  def my_all?
+    each do |i|
+      return puts false unless yield i
+    end
+    puts true
+  end
+
+  def my_any?
+    each do |i|
+      return true if yield i
+    end
+    false
+  end
```

Comment:
Kindly rewrite this method to make it pass the tests respecting the ones run with `my_all?` up there.